c2corg_ui/templates/article/edit.html

```diff
@@ -0,0 +1,601 @@
+<%!
+from c2corg_common.attributes import default_langs, activities, condition_ratings, \
+    quality_types, hut_status, lift_status, frequentation_types, avalanche_signs, \
+    route_duration_types, access_conditions, glacier_ratings
+%>
+
+<%inherit file="../base.html"/>
+
+<%
+updating_doc = article_id and article_lang
+%>
+
+<%namespace file="../helpers/common.html" import="show_title"/>
+<%namespace file="../helpers/edit.html" import="show_editing_associated_waypoints,
+    show_editing_associated_routes,show_preview_container,
+    show_editing_buttons"/>
+
+<%block name="pagetitle"><title ng-init="id=${article_id if article_id else 0}" ng-bind="id ? mainCtrl.page_title('Editing an article') : mainCtrl.page_title('Creating an article')">${show_title('Create/edit article')}</title></%block>
+<%block name="metarobots"><meta name="robots" content="noindex,nofollow"></%block>
+
+<section class="create-edit-document">
+
+  <h1 ng-cloak ng-init="id=${article_id if article_id else 0}"
+      ng-bind="id ? ('Editing an article' | translate) : ('Creating an article' | translate)"
+      class="text-center"></h1>
+
+  <form app-loading app-document-editing="articles" app-document-editing-model="article"
+    app-document-editing-controller-name="apparticleEditingController"
+  % if updating_doc:
```

Comment:
I wouldn't. Since we have not done it for other document types editing forms.
In addition it's related to an attribute within an HTML element, it's not 2 elements embedded one in another.